modules/util.py

```diff
@@ -394,15 +396,20 @@ def find_shadow_chrome_element(
         self, nodes: list[WebElement], selector: list
     ) -> Union[WebElement, None]:
         logging.info("Selecting element in Chrome Context Shadow DOM...")
-        if selector[0] != By.ID:
+        if selector[0] not in self.allowed_selectors:
+            # if selector[0] != By.ID:
             raise ValueError(
-                "Currently shadow elements in chrome can only be selected by ID."
+                "Currently shadow elements in chrome can only be selected by ID and classname."
```

Comment:
Didn't we create functionality for selecting shadow chrome elements by tag name? What happened to that code?

Reply:
i think i discarded the changes bc it wasn't relevant atm to the shadow element. in the end, it was for the pin tab PR and that was figured out with a workaround

Reply:
OK, that's fine